packages/server/types/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "baseUrl": "../",
     "forceConsistentCasingInFileNames": true,
     "jsx": "react",
-    "lib": ["es6"],
+    "lib": ["es6", "dom"],
```

Comment:
q: why do we need to a DOM lib to `@emotion/server` here? 🤔

Reply:
See my 2nd bullet point in the original comment 😉